### PR TITLE
[#130020741] Implement automation of credential rotation (continued)

### PR DESCRIPTION
**Pull request by mtekel**
## What

[Implement automation of credential rotation (continued)](https://www.pivotaltracker.com/story/show/130020741)

Add tasks to clean credentials we deem safe to rotate. 
Moved from paas-cf, see https://github.com/alphagov/paas-cf/pull/802

## How to review

Run the tasks to clear bosh and concourse credentials. Run the main pipeline to generate new creds and apply them. All should work fine.

## Who can review

not @mtekel

### CODE REVIEW

File: concourse/pipelines/create.yml
Comment (dcarley) [inline concourse/pipelines/create.yml:1208]: > Fix script so that it doesn't "swallow" exit code.

Could you explain in the commit message what the effect of this is before and after?

**Review by dcarley — CHANGES_REQUESTED** [review]
Works. One small request about commit messages. Good to merge once updated.

File: concourse/pipelines/create.yml
Comment (mtekel) [reply concourse/pipelines/create.yml:1208]: It was failing shellcheck. I have no idea how was this passing before. I've updated commit message.

Comment (alext) [reply concourse/pipelines/create.yml:1208]: It might have been always failing shellcheck. I'm not sure that we've made the warnings generated by pipecleaner fail the build (we're not passing the `--fatal-warnings` flag to pipecleaner). We probably should now that the last one has been fixed.

Comment (mtekel) [reply concourse/pipelines/create.yml:1208]: Added fatal warnings.

Comment (dcarley) [reply concourse/pipelines/create.yml:1208]: Thanks 👍

Comment (dcarley) [reply concourse/pipelines/create.yml:1208]: PS: I think that should have been a separate commit, but not important enough to hold up the merge.

**Review by dcarley — APPROVED** [review]

**Pull request merged**
